docs/reference/api/storage/Dir.md

```diff
@@ -2,12 +2,21 @@
 
 <span class="images">![](https://os-doc-builder.test.mbed.com/docs/development/mbed-os-api-doxy/classmbed_1_1_dir.png)<span>Dir class hierarchy</span></span>
```

Comment:
Do we need this @AnotherButler ?

Reply:
@kegilbert Good question. Yes, this will render correctly.